ui/app/products/components/pnc-products-data-table/pncProductsDataTable.js

```diff
@@ -50,6 +52,25 @@
           filterType: 'text'
         }
       ];
+
+      $ctrl.productsSortingFields = [{
+        id: 'name',
+        title: 'Name'
+      },
+      {
+        id: 'abbreviation',
+        title: 'Abbreviation'
+      },
+      {
+        id: 'productCode',
+        title: 'Product Code'
+      },
+      {
+        id: 'pgmSystemName',
```

Comment:
These just need to be updated to remove the fields that are being removed in 2.0